src/features/user/Account/AccountHistory.module.scss

```diff
@@ -679,6 +686,10 @@
     display: none;
   }
 
+  .issueButtonMobileContainer {
+    display: none;
+  }
+
   .modal {
```

Comment:
Lines 689-692 and 718-721 are duplicate

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1677/commits/91ef6074e08df526d022b99bb162ebb7c06d8cd3